src/utils/datetime/formatters.test.ts

```diff
@@ -64,6 +65,12 @@ describe('the DateTime formatter', () => {
   }
 
   describe('formatting DateTimes in UTC', () => {
+    it('formats DateTimes in the default time YYYY-MM-DD in UTC', () => {
```

Comment:
```suggestion
    it('formats DateTimes in the format YYYY-MM-DD in UTC', () => {
```
also I think for organization sake, keep the default case at the top of the file

Reply:
For sure! Done 👍